lang/lang_en_fr.txt

```diff
@@ -476,6 +530,10 @@
 "Live adjust Z"
 "Ajuster Z en dir."
 
+#c=20 r=6
+"Insert the filament (do not load it) into the extruder and then press the knob."
+"Veuillez inserer le filament (ne pour le charger) dans l'extrudeur, puis appuyez sur le bouton."
+
```

Comment:
Veuillez inserer le filament ( ne le chargez pas) dans l'extrudeur, puis appuyez sur le bouton

Reply:
> Veuillez inserer le filament ( ne le chargez pas) dans l'extrudeur, puis appuyez sur le bouton

Use this.